packages/front-end/pages/sdks/SdkWebhooks.tsx

```diff
@@ -52,53 +54,54 @@ export default function SdkWebhooks({ sdkid }) {
             <em>never fired</em>
           )}
         </td>
-        {hasWebhookPermissions && (
-          <td>
-            <Button
-              color="link"
-              className="btn-sm"
-              onClick={async () => {
-                await apiCall(`/webhook/test/${webhook.id}`, {
-                  method: "get",
-                });
-                mutate();
-              }}
-            >
-              <BsArrowRepeat /> Test Webhook
-            </Button>
-          </td>
-        )}
         <td>
-          {hasWebhookPermissions && (
-            <>
-              <div className="col-auto">
-                <MoreMenu>
-                  <button
-                    className="dropdown-item"
-                    onClick={(e) => {
-                      e.preventDefault();
-                      if (!disableWebhookCreate)
-                        setCreateWebhookModalOpen(webhook);
-                    }}
-                  >
-                    Edit
-                  </button>
-                  <DeleteButton
-                    className="dropdown-item"
-                    displayName="SDK Connection"
-                    text="Delete"
-                    useIcon={false}
-                    onClick={async () => {
-                      await apiCall(`/webhook/${webhook.id}`, {
-                        method: "DELETE",
-                      });
-                      mutate();
-                    }}
-                  />
-                </MoreMenu>
-              </div>
-            </>
-          )}
+          <Button
+            color="link"
+            className="btn-sm"
+            disabled={
+              !canCreateWebhooks || !canUpdateWebhook || !canDeleteWebhook
```

Comment:
Let's just use `canUpdateWebhook` for this.  We could create a new permission for testing webhooks, but that's probably overkill.  If they can update a webhook, then they should be able to test it.